src/app/auth/sign-up/sign-up.component.html

```diff
@@ -0,0 +1,42 @@
+<form [formGroup]="signUpForm"
+  (ngSubmit)="onSubmit()">
+  <mat-form-field>
+    <mat-label>Username</mat-label>
+    <input matInput
+      formControlName="displayName"
+      name="displayName"
+      type="text"
+      required />
```

Comment:
please write attributes either in one line or each attribute on a new line

Reply:
Done